components/DeleteButton/index.js

```diff
@@ -1,17 +1,21 @@
 import styled from "styled-components";
 
-const StyledButton = styled.button`
+const DeleteBtn = styled.button`
```

Comment:
I recommend to not use abbreviations as your code will be easier to read without them 🙂 
```suggestion
const DeleteButton = styled.button`
```